andela/ah-frontend-zeus
# #162163132 Users should be able to Login via Google or Facebook

## What does this PR do?
- The PR ensures that users can log in using their favourite social media platform.
### Description of task to be completed?
- Users should be able to Login via Google or Facebook.
### How should this be manually tested?
- Clone the repository.
- Install the dependencies using `yarn install`.
- Start the application using `yarn start`.
- A user should have a facebook or google account to sign in.
- Click the respective buttons to login via Facebook or Google.
<img width="1315" alt="screen shot 2019-01-22 at 10 37 41" src="https://user-images.githubusercontent.com/31479458/51519377-dce90400-1e31-11e9-81db-d8325d74c12e.png">


 ## Login with Facebook.
<img width="469" alt="screen shot 2019-01-18 at 07 50 10" src="https://user-images.githubusercontent.com/31479458/51368024-7648ab00-1afe-11e9-83b0-51f453f990d6.png">

## Login with Google.
<img width="517" alt="screen shot 2019-01-18 at 07 48 08" src="https://user-images.githubusercontent.com/31479458/51368041-86608a80-1afe-11e9-8252-b63d71d85642.png">

### What are the relevant pivotal tracker stories?
[#162163132](https://www.pivotaltracker.com/story/show/162163132)
